### PR TITLE
Update wording in "cancel question" modal

### LINKED ISSUE #140: Rework "cancel question" modal
Simeng told me that the phrasing and coloring on the "cancel question" model is pretty confusing, and I agree. There are two buttons, "Cancel question" and "Cancel", and "Cancel question" is the danger red color. It's also not immediately clear what it even means to cancel a question.

### PR DESCRIPTION
**Pull request by Xiangmingchen**
When a staff cancel answering a question, the messages on the button and modal are more clear. Fixes #140

### CODE REVIEW

**Comment by Xiangmingchen** [comment]
which one is the linter issue? Sure I will change the text on button

**Comment by nwalters512** [comment]
Click on the Travis status check and read through for errors.

**Comment by Xiangmingchen** [comment]
There seems to be test cases for the button with message “cancel”, should I change those test cases as well?

**Comment by Xiangmingchen** [comment]
I think I accidentally added yarn to the dependency. I will remove that

**Comment by nwalters512** [comment]
Almost there! The text that a person sees when they click cancel but aren't the one answering is a bit confusing; it's the same "You will stop answering this question but this question will stay in the queue." that a person who *is* answering will see. Can you look into improving that?

**Comment by Xiangmingchen** [comment]
Do you know how can I sign in when running locally as another user? Both as a non admin staff and as a student, so that I can see the result of my changes.

**Comment by nwalters512** [comment]
Check the Readme: https://github.com/illinois/queue#multiple-users-in-dev-mode

**Comment by Xiangmingchen** [comment]
Does "Someone else is answering this question and you will interrupt it. This question will stay in the queue." sound good? The confirm button text will be "interrupt" and the cancel button will say "cancel"

**Comment by Xiangmingchen** [comment]
#140

File: src/components/Question.js [hunk 1]
```diff
@@ -29,18 +29,24 @@ class Question extends React.Component {
...
-              <Button
-                color="primary"
-                className="mr-2"
-                onClick={() => this.props.finishedAnswering(id)}
-              >
-                Finish Answering
-              </Button>
-            )}
             <Button color="light" onClick={() => this.props.cancelQuestion()}>
               Cancel
```
Comment (nwalters512) [inline src/components/Question.js]: I think this should read "Interrupt" to match the confirm text in the modal.

**Comment by nwalters512** [comment]
@Xiangmingchen can you fix the newly broken test?

**Comment by Xiangmingchen** [comment]
Thanks for reminding, I have updated the Changelog.

**Pull request merged**